ckan_cloud_operator/providers/ckan/instance/manager.py

```diff
@@ -349,7 +349,21 @@ def create_ckan_admin_user(instance_id_or_name, name, email=None, password=None,
         'password': password
     }
     if not dry_run:
-        deployment_manager.create_ckan_admin_user(instance_id, instance_type, instance, user)
+        pod_name = _get_running_pod_name(instance_id)
+        name, password, email = [user[k] for k in ['name', 'password', 'email']]
+        logs.info(f'Creating CKAN admin user with {name} ({email}) and {password} on pod {pod_name}')
```

Comment:
Is it safe to print the password in the logs?

Reply:
Passwords are autogenerated if not  passed. User should somehow know password if so. But maybe we don't need that part and it should be required?

Reply:
I think the workflow is okay. I just don't like the idea of passwords being printed in logs since it can be a security risk. Same with passing passwords as parameters in CLI tools since it can be recovered from the command line history. 

For now, I think that just not logging the password in the `logs.info()` is enough. But a good improvement will be the user to enter the password twice in an interactive console when the user executes the command.